react/components/SponsoredBanner/__test__/SponsoredBanner.test.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { render } from '@vtex/test-tools/react'
+
+import { SponsoredBanner } from '../SponsoredBanner'
+
+test('greets Fred', () => {
+  const { queryByText } = render(<SponsoredBanner />)
+
+  expect(queryByText('Hey, Fred')).toBeInTheDocument()
+})
```

Comment:
I think we can delete this file, right?

Reply:
Actually I'm going to update this to create unit tests.